src/policy/CMSRucioPolicy/permission.py

```diff
@@ -232,17 +233,17 @@ def _get_rule_size(rules):
 
     # Check if the account is banned
     if has_account_attribute(account, 'auto_approve_banned', session=session):
-        return False
+        return PermissionResult(False, "The account is banned.")
```

Comment:
I would add instructions for further action - such as creating a ticket with DM in jira.